modules/monitoring-resolving-the-kubepersistentvolumefillingup-alert-firing-for-prometheus.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * monitoring/troubleshooting-monitoring-issues.adoc
+// * support/troubleshooting/investigating-monitoring-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="resolving-the-kubepersistentvolumefillingup-alert-firing-for-prometheus_{context}"]
+= Resolving the KubePersistentVolumeFillingUp alert firing for Prometheus
+
+As a cluster administrator, you can resolve the `KubePersistentVolumeFillingUp` alert being triggered for Prometheus. 
+
+The critical alert fires when a persistent volume (PV) claimed by a `prometheus-k8s-*` pod in the `openshift-monitoring` project has less than 3% of its total space remaining. This can cause Prometheus to not function normally.
+
+[NOTE]
+====
+There are two `KubePersistentVolumeFillingUp` alerts:
+
+* *Critical alert*:  The alert with the `severity="critical"` label is triggered when the mounted PV has less than 3% of its total space remaining.
+* *Warning alert*: The alert with the `severity="warning"` label is triggered when the mounted PV has less than 15% of its total space remaining and is expected to fill up within four days.
+====
+
+To address this issue, you can remove Prometheus time series database (TSDB) blocks to create more space for the PV.
+
+.Prerequisites
+
+ifndef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `dedicated-admin` role.
+endif::openshift-dedicated,openshift-rosa[]
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. List the size of all TSDB blocks, sorted from oldest to newest:
```

Comment:
This step needs some explicit phrasing about running a command, per our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command):
```suggestion
. List the size of all TSDB blocks, sorted from oldest to newest, by running the following command:
```